web/template/layout/mlpa-case-summary.gohtml

```diff
@@ -0,0 +1,60 @@
+                        <span class="case-summary-toggle-text">Hide</span>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="govuk-grid-row case-summary">
+            <div class="govuk-grid-column-one-third">
+                <div class="govuk-!-display-block">
+                    <svg class="app-!-svg-icon" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" height="20px" width="20px" overflow="visible">
+                        <circle cx="9.5" cy="9.5" r="9.5" fill="#1D70B8"/>
+                        <circle cx="9.5" fill="#fff" cy="13.8" r="1.3"/>
+                        <path fill="#fff" d="M8.4 4.5h2.2l-.2 7H8.6l-.2-7Z"/>
+                    </svg>
+                    <h3 class="govuk-heading-s govuk-!-display-inline">Warnings</h3>
+                </div>
+                <p class="govuk-body">There are none in place.</p>
+                <div class="govuk-!-display-block">
+                    <svg class="app-!-svg-icon" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" height="20px" width="20px" overflow="visible">
+                        <circle cx="9.5" cy="9.5" r="9.5" fill="#1D70B8"/>
+                        <circle cx="9.5" fill="#fff" cy="13.8" r="1.3"/>
+                        <path fill="#fff" d="M8.4 4.5h2.2l-.2 7H8.6l-.2-7Z"/>
+                    </svg>
+                    <h3 class="govuk-heading-s govuk-!-display-inline">Complaints, objections & investigations</h3>
+                </div>
+                <p class="govuk-body">There are none in place.</p>
+            </div>
+            <div class="govuk-grid-column-two-thirds">
+                <table class="govuk-table">
+                    <thead class="govuk-table__head app-!-table-head__no-vertical-padding">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header">
+                                <svg class="app-!-svg-icon govuk-!-display-inline" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" height="20px" width="20px" overflow="visible">
+                                    <circle cx="9.5" cy="9.5" r="9.5" fill="#1D70B8"/>
+                                    <path d="m6 10 2 2 5-5" stroke="#fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                                </svg>
+                                Tasks
+                            </th>
+                            <th scope="col" class="govuk-table__header">Due date</th>
+                            <th scope="col" class="govuk-table__header">Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody class="govuk-table__body">
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell app-!-table-row__no-border">There are no live tasks.</td>
```

Comment:
You can make the cell span across three columns to ensure it doesn't wrap too early
```suggestion
                            <td class="govuk-table__cell app-!-table-row__no-border" colspan="3">There are no live tasks.</td>
```